liqo-peering/values.yaml

```diff
@@ -0,0 +1,20 @@
+role: consumer  # or provider
+
+clusterId: local
+privateWireguardKey: # placeholder for private wireguard key
+publicWireguardKey: # placeholder for public wireguard key
+
+remoteCluster: 
+  clusterId: remote
+  userCommonName: "user@cluster"  # The CN of the user in the certificate
```

Comment:
I'd prefer not to have fake values in the chart base values.yaml, better to have it empty and force definition (in our case in gitops).